lib/deck.rb

```diff
@@ -1,7 +1,7 @@
 require_relative 'card'
 
+# Represents a deck of cards
```

Comment:
Do we need some more detail here?

This seems like it was done just to make rubocop shut up.